hawkbit-rest/hawkbit-mgmt-api/src/main/java/org/eclipse/hawkbit/mgmt/json/model/action/MgmtActionRequestBodyPut.java

```diff
@@ -19,14 +19,14 @@
 public class MgmtActionRequestBodyPut {
 
     @JsonProperty
-    private MgmtActionType forceType;
+    private MgmtActionType actionType;
```

Comment:
This is totally correct but I'm afraid that will break the management API. Sorry that I haven't seen this before.
Maybe it is sufficient to just add `@JsonProperty(value="forceType")` and leave the change as it is